### PR TITLE
[apm serverless] Add examples using `trace_continuation_strategy` to sampling docs

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Add changes introduced in https://github.com/elastic/observability-docs/pull/4167 to the serverless docs.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-mydf3cf67-elastic-dev.vercel.app

* Built with commit 0c621d8db2996391a19b5ba6bd3f5f04e033cf8b

> Issues? Visit #next-docs in Slack

File: docs/en/serverless/apm/apm-transaction-sampling.mdx [hunk 1]
```diff
@@ -28,23 +28,69 @@ Head-based sampling is quick and easy to set up.
...
-### Distributed tracing with head-based sampling
+### Distributed tracing
 
 In a _distributed_ trace, the sampling decision is still made when the trace is initiated.
 Each subsequent service respects the initial service's sampling decision, regardless of its configured sample rate;
 the result is a sampling percentage that matches the initiating service.
 
-In this example, `Service A` initiates four transactions and has sample rate of `.5` (`50%`).
-The sample rates of `Service B` and `Service C` are ignored.
+In the example in _Figure 1_, `Service A` initiates four transactions and has sample rate of `.5` (`50%`).
```
Comment (dedemorton) [inline docs/en/serverless/apm/apm-transaction-sampling.mdx:37]:
```suggestion
In the example in _Figure 1_, `Service A` initiates four transactions and has a sample rate of `.5` (`50%`).
```

**Review by dedemorton — COMMENTED** [review]
So minor it's hardly worth mentioning...

**Review by dedemorton — APPROVED** [review]
LGTM!

**Pull request merged**